rflx/generator/core.py

```diff
@@ -168,63 +270,66 @@ def __process_message(self, message: Message, seen_types: Set[str]) -> None:
         self.__create_message_unit(message)
 
         for field_type in message.types.values():
-            if not is_seen_type(f'{message.package}.{field_type}', seen_types):
+            if not is_seen_type(f"{message.package}.{field_type}", seen_types):
                 self.__create_type(field_type, message.package)
 
     def __process_derived_message(self, message: DerivedMessage, seen_types: Set[str]) -> None:
         if message.package not in self.units:
             self.__create_unit(
-                message.package,
-                [WithClause(f'{self.prefix}{message.base_package}')],
-                True)
+                message.package, [WithClause(f"{self.prefix}{message.base_package}")], True
+            )
 
         self.__create_message_unit(message)
 
         for field_type in message.types.values():
-            if not is_seen_type(f'{message.package}.{field_type.name}', seen_types):
+            if not is_seen_type(f"{message.package}.{field_type.name}", seen_types):
                 self.__create_subtype(field_type, message.package, message.base_package)
 
     def __create_unit(self, package_name: str, context: List[ContextItem], top_level: bool) -> None:
-        name = f'{self.prefix}{package_name}'
+        name = f"{self.prefix}{package_name}"
         self.units[package_name] = PackageUnit(
             self.__common_context() + context if top_level else context,
             PackageDeclaration(name),
-            PackageBody(name))
+            PackageBody(name),
+        )
 
-    def __create_generic_message_unit(self, message: Message) -> None:  # pylint: disable=too-many-statements
+    # pylint: disable=too-many-statements
+    def __create_generic_message_unit(self, message: Message) -> None:
         context: List[ContextItem] = [
```

Comment:
This changes moves the pylint comment from inside the function to before the function, does this affect the comments scope?

Reply:
This pylint comments can placed on the same line (as before) or before the line (like after the change). So it is not a semantic change.